app/assets/stylesheets/core/_link.scss

```diff
@@ -1,6 +1,7 @@
 // TODO: support this modifier in govuk-frontend
 .govuk-link.app-link--destructive {
   color: $govuk-error-colour;
+  @include govuk-font(19);
```

Comment:
I'm surprised this wasn't an issue when the button is used in other contexts i.e. on the images index page. How come it's the right size over there?

Reply:
This is only for destructive buttons which are not widely used, maybe that's why we haven't got them in a context that didn't have a font-size set.